src/SegmentContentAccessCheck.php

```diff
@@ -79,6 +103,12 @@ public function checkAccess(EntityInterface $entity, $operation, AccountInterfac
 
       if (in_array($bundle, $permissions_config[$type])) {
 
+        $shared_content_client = $this->privateTempStore->get($this->request->getClientIp());
+        if ($shared_content_client) {
+          // Bypass permissions for shared content client servers.
+          return AccessResult::neutral();
```

Comment:
```suggestion
          return AccessResult::allowed();
```
I think we should use allow. When we will update permissions, neutral means that it will be unavailable for anonymous and neutral means deny.

Reply:
Fixed